src/Uno.Extensions.Logging/LoggingBuilderExtensions.cs

```diff
@@ -1,7 +1,22 @@
 ﻿namespace Uno.Extensions;
 
+/// <summary>
+/// Extension methods to adjust the scope of collected logs. (log level)
+/// </summary>
 public static class LoggingBuilderExtensions
 {
+	/// <summary>
+	/// Sets the log level for the Uno Platform core namespaces.
+	/// </summary>
```

Comment:
Add the list of namespaces that this method sets the filter for in the comments (same for the other LogLevel methods)

Reply:
Added with 127326d40fafec920e57fd486412ace4bed15ff9